multiversx_sdk_cli/utils.py

```diff
@@ -37,9 +37,13 @@ def to_json(self):
 
 
 class BasicEncoder(json.JSONEncoder):
-    def default(self, o: Any):
+    def default(self, o: Any) -> Any:
         if isinstance(o, ISerializable):
             return o.to_dictionary()
+        if isinstance(o, bytes):
```

Comment:
Generally speaking, is not a breaking change (e.g. it does not alter the output for existing flows that use --outfile or so).